playground/integration_test/playground_flow_test.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:demo_ui_components/demo_ui_components.dart';
+import 'package:flutter/material.dart';
+import 'package:patrol/patrol.dart';
+import 'package:playground/main.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  patrolTest(
```

Comment:
We aim to minimize the use of third-party libraries in our project to make it easier for new contributors to join. Can you please re-write without patrol.
We will provide a test template soon that explains our approach and style.

Reply:
Ah thank you i will handle with core test libs